be/src/runtime/memory/thread_mem_tracker_mgr.h

```diff
@@ -193,7 +198,8 @@ inline void ThreadMemTrackerMgr::consume(int64_t size, int skip_large_memory_che
             _reserved_mem -= size;
             // store bytes that not synchronized to process reserved memory.
             _untracked_mem += size;
-            if (_untracked_mem >= SYNC_PROC_RESERVED_INTERVAL_BYTES) {
+            // If _reserved_mem is added back, it still need synchronized to process reserved_memory.
+            if (std::abs(_untracked_mem) >= SYNC_PROC_RESERVED_INTERVAL_BYTES) {
```

Comment:
add comment here to explain why this value will < 0

Reply:
done